packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md

```diff
@@ -316,3 +334,57 @@ func retrievePushChallenges(accessToken: String) {
 ```
 
 The code assumes that each enrollment uses the same access token. Use the appropriate access token if your app handles multiple user accounts.
+
+Alternatively, you can retrieve undelivered challenges using the [MyAccount App Authenticators API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/AppAuthenticator/#tag/AppAuthenticator/operation/listAppAuthenticatorPendingPushNotificationChallenges).
+
+## Access token management
+
+> **Note:** To enable the JWT bearer grant type:
+>  * Send a PUT request to `/apps/{appId}`. Ensure that the `grant_types` array contains the following string:
+>    `urn:ietf:params:oauth:grant-type:jwt-bearer`
+>  * If you use custom authorization servers, update the policy rules to update the grant type.
```

Comment:
I know the link to the API is already in the nut facts. Should it be added in this note blurb as well?

Reply:
Yes, I agree. It would be helpful to have a link to the `PUT /apps/${appId}/` endpoint description.